src/mono_font/mono_text_style.rs

```diff
@@ -1170,4 +1154,30 @@ mod tests {
         );
         assert_eq!(iter.next(), Some((Point::new(38, 20), LineElement::Done)));
     }
+
+    #[test]
+    fn builder_change_font() {
+        let _style = {
+            let font = MonoFont {
+                image: ImageRaw::new_binary(&[1, 2, 3], 1),
+                character_size: Size::new(1, 2),
+                character_spacing: 0,
+                baseline: 0,
+                strikethrough: DecorationDimensions::default_strikethrough(2),
+                underline: DecorationDimensions::default_strikethrough(2),
```

Comment:
Should this be?

```suggestion
                underline: DecorationDimensions::default_underline(2),
```

Doesn't matter much for a test but probably good for consistency/regression catching.

Reply:
Good catch.